.github/workflows/rector.yml

```
@@ -0,0 +1,37 @@
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  rector:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)